src/evaluation/WorkspaceManager.jl

```diff
@@ -81,8 +81,12 @@ function create_emptyworkspacemodule(pid::Integer)::Symbol
     new_workspace_name
 end
 
-function create_workspaceprocess()::Integer
-    pid = Distributed.addprocs(1) |> first
+function create_workspaceprocess(;project::String="")::Integer
+    if isempty(project)
+        project = default_env()
+    end
+    # NOTE: notebook environment should not be the same as server process environment
```

Comment:
Sorry, I might be too brief in the note, I wrote this to explain why I force all workspace process to start with a `--project` flag so it will never inherit server process environment implicitly. 

I think this behaviour is more natural, since in principal we don't assume different notebook shares the same environment at the backend (unless it is declared explicitly in the frontend).

Reply:
The other solution is to inherit the environment if `project === nothing` is `true`. But I find this is not something I'd intuitively expect when I work with notebook

Reply:
Does it currently inherit the environment? That's not good

Reply:
you mean `master` branch or this PR? 

1. it inherits from the environment in `master`
2. it does not inherit in this PR

I was just wondering if I should preserve `master` branch behaviour (since it might break things if not), but then I realize it's not intuitive to inherit the environment.

Reply:
No that was unintended! It should not inherit - thanks for spotting the mistake